pkg/controller/clusterconnection/clusterconnection_controller.go

```diff
@@ -331,16 +331,29 @@ func (r *ReconcileConnection) Reconcile(ctx context.Context, request reconcile.R
 		}
 	}
 
+	// Populate a list of namespaces to be displayed in the service graph Tigera infrastructure layer.
+	sgLayerTigeraNameSpaces := render.DefaultSGLayerTigeraNamespaces()
+	if !sgLayerTigeraNameSpaces[render.GuardianNamespace] {
```

Comment:
I think there is no need for the if statement and we can simply set it to true or am I missing something?

Reply:
I think this applies to all of these if-statements in the PR.

Reply:
Done